pkg/framework/jobs.go

```diff
@@ -0,0 +1,63 @@
+package framework
+
+import (
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+)
+
+func NewWorkLoad(njobs int32, memoryRequest resource.Quantity, workloadJobName string,
```

Comment:
Nit, this should have some sort of comment
```suggestion
// NewWorkload generates a batchv1.Job to deploy a pod which will sleep for one day.
// This can be used to add workloads to the cluster to affect scheduling and excercise
// autoscaling within tests.
func NewWorkLoad(njobs int32, memoryRequest resource.Quantity, workloadJobName string,
```